R/process_lookup_postcode.R

```diff
@@ -4,18 +4,19 @@
 #' postcode lookup, it will return the final data
 #' but also write this out as a zsav and rds.
 #'
+#' @param file_path Path to spd lookup.
 #' @param write_to_disk (optional) Should the data be written to disk default is
 #' `TRUE` i.e. write the data to disk.
 #'
 #' @return the final data as a [tibble][tibble::tibble-package].
 #' @export
 #' @family process extracts
 
-process_lookup_postcode <- function(write_to_disk = TRUE) {
+process_lookup_postcode <- function(file_path = get_spd_path(), write_to_disk = TRUE) {
```

Comment:
The simd and locality files should also be parameters (and therefore targets). This function should be re-run if any files change on disk, so we need to have them all as targets.